src/transport/BUILD.gn

```diff
@@ -50,6 +50,7 @@ static_library("transport") {
     "${chip_root}/src/lib/core",
     "${chip_root}/src/lib/support",
     "${chip_root}/src/platform",
+    "${chip_root}/src/setup_payload",
```

Comment:
I'm suspicious of this dependency, should transport really depend on the code to generate a QR code?

Maybe the needed constants should moved or split off.

Reply:
I agree that it sounds odd. One solution could be to move `Rendezvous*` inside its own folder. It would make sense for Rendezvous to depends on the setup payload.
I'm fine with a followup for that for what it worth.

Reply:
Yeah I completely agree as well. I was initially contemplating setting a Range on the `RendezvousParams`object. i.e set a max discriminator value to remove this payload dependency. wdygt?

Reply:
For now I've added a constant and filed #3631